cypress/support/step_definitions/event.js

```diff
@@ -5,3 +5,24 @@ When('I click on {string}', (templateSelector) => {
   const selector = nunjucks.renderString(templateSelector, cy.context);
   cy.get(selector).click();
 });
+
+When('I double click on last element of {string}', (templateSelector) => {
+  const selector = nunjucks.renderString(templateSelector, cy.context);
+  cy.get(selector).last().dblclick();
+});
+
+When('I double click on elements of {string}', (templateSelector) => {
+  const selector = nunjucks.renderString(templateSelector, cy.context);
+  cy.get(selector).click({ multiple: true }).dblclick({ multiple: true });
+});
+
+When('I double click on {string}', (templateSelector) => {
+  const selector = nunjucks.renderString(templateSelector, cy.context);
+  cy.get(selector).dblclick();
+});
+
+When('I click on active tab element of {string}', (templateSelector) => {
```

Comment:
remove this rules, because I prefere click on the active element with the data-cy selection.
I expect to have this data-cy for active tab: `[data-cy=\"active-tab\"] [data-cy=\"close-file-tab\"]` or something like this

Reply:
done